futures/tests/stream_futures_unordered.rs

```diff
@@ -340,7 +340,7 @@ fn polled_only_once_at_most_per_iteration() {
 
     let mut tasks = FuturesUnordered::from_iter(vec![F::default(); 33]);
     assert!(tasks.poll_next_unpin(cx).is_pending());
-    assert_eq!(33, tasks.iter().filter(|f| f.polled).count());
+    assert_eq!(32, tasks.iter().filter(|f| f.polled).count());
```

Comment:
Shouldn't this test be adjusted to ensure that eventually all of the contained futures are polled?

Reply:
The purpose of the test is to make sure that the same future is not polled multiple times in a single poll_next call (https://github.com/rust-lang/futures-rs/pull/2333), so I think it's fine as is.

https://github.com/rust-lang/futures-rs/blob/d3747e688a0356a03a45ad7ea9bec95e6381ad9c/futures/tests/stream_futures_unordered.rs#L316